libr/include/r_cons.h

```diff
@@ -142,9 +142,12 @@ enum {
 
 enum { COLOR_MODE_DISABLED = 0, COLOR_MODE_16, COLOR_MODE_256, COLOR_MODE_16M };
 
-enum { ALPHA_RESET = 0x00, ALPHA_FG = 0x01, ALPHA_BG = 0x02, ALPHA_FGBG = 0x03, ALPHA_BOLD = 0x04 };
+enum { ALPHA_RESET = 0x00, ALPHA_FG = 0x01, ALPHA_BG = 0x02, ALPHA_FGBG = 0x03 };
+enum { TERM_ATTR_BOLD = 1 << 1 };
```

Comment:
Why an enum if you have only one value? What about TERM_ATTR_ITALIC, TERM_ATTR_UNDERLINE, ...?
Haha

Reply:
TERM_ATTR_ITALIC ... are not used much so I do not create an enum constant for them.

Reply:
Meh ok

Reply:
yeah will be good to have them prefixed by RColor or R_CONS_ATTR_*